nautobot/core/settings.yaml

```diff
@@ -1058,23 +1058,16 @@ properties:
       +/- 2.0.0
           This directory no longer requires an `__init__.py` file.
     environment_variable: "NAUTOBOT_JOBS_ROOT"
-  KUBECONFIG:
+  KUBERNETES_JOB_POD_HOST:
     default: ""
-    description: "File path to the configuration file used to spin up a Kubernetes pod."
-    environment_variable: "NAUTOBOT_KUBECONFIG"
+    description: "HTTP address of the Kubernetes cluster."
+    environment_variable: "NAUTOBOT_KUBERNETES_JOB_POD_HOST"
     type: "string"
     version_added: "2.4.0"
-  KUBERNETES_JOB_CONTAINER_NAME:
-    default: ""
-    description: "The nautobot container name running on the Kubernetes pod."
-    environment_variable: "NAUTOBOT_KUBERNETES_JOB_CONTAINER_NAME"
-    type: "string"
-    version_added: "2.4.0"
-  KUBERNETES_JOB_IMAGE_NAME:
-    default: ""
-    description: "The name of the docker image used to build the nautobot container running on the Kubernetes pod."
-    environment_variable: "NAUTOBOT_KUBERNETES_JOB_IMAGE_NAME"
-    type: "string"
+  KUBERNETES_JOB_POD_MANIFEST:
+    default: {}
+    description: "A dictionary that contains the pod manifest used to create a job pod in the kubernetes cluster."
```

Comment:
We should *probably* plan to add a new section in the Job docs with details of how to set this up. Not replicating the k8s documentation or anything to that level of detail, but more guidance on how, given you have a k8s setup, to configure Nautobot to make use of it.